src/plugins/__tests__/bootstrap.test.js

```diff
@@ -0,0 +1,22 @@
+let Colonel   = require('../../Colonel')
+let bootstrap = require('../bootstrap')
+
+describe('bootstrap plugin', function() {
+
+  it ('can serialize an input', function() {
+    let app   = new Colonel({ el: document.createElement('div') })
+    let input = document.createElement('textarea')
+
+    input.value = JSON.stringify([{
+      blocks  : [],
+      content : {},
+      type    : 'section'
+    }])
+
+    bootstrap.register(app, { blocks: input, blockTypes: [] }, function() {
+      app.refine('blocks').first().should.have.property('type', 'section')
```

Comment:
Good point. Although `bootstrap` isn't async, it might be in the future. I'll go ahead and use done() here